### DISCUSSION TITLE
Please welcome Jeff Gehlbach as NetBox product manager!

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#14464] [Announcements]
I'm excited to announce a new addition to the NetBox team: Jeff Gehlbach (@jeffgdotorg)! Jeff was recently hired as a Product & Program Manager at NetBox Labs, and will be responsible for helping guide the long-term strategy of both the open source project and our commercial products.

An open source veteran, Jeff was previously product manager for the OpenNMS Horizon ecosystem. Over the years his roles have included customer support, network engineering, *NIX sysadmin, software development, and service delivery, with an emphasis on open-source monitoring and observability. 

You'll begin to see Jeff engaging right here on GitHub very soon, as he works to help triage issues, capturing valuable community feedback and use cases to prioritize development work and drive our long-term roadmaps. Welcome Jeff!

**Comment by network2501** [comment]
Congrats and welcome Jeff!

**Comment by jeffgdotorg** [comment]
Thanks, Jeremy! I’m excited to be joining the project and the company, and looking forward to getting to know the community.

**Comment by reissmann** [comment]
Jeff, it's great to see that you've moved from one awesome open source project to another awesome open source project. I wish you a great start!

**Reply by jeffgdotorg** [reply]
Hallo Sven, it’s nice to bump into you here! Thank you for the warm welcome, and I hope to see you again soon IRL. At FOSDEM perhaps?
